examples/basic/pages/admin/[[...slug]].js

```diff
@@ -0,0 +1,3 @@
+import { TinaAdmin } from 'tinacms'
```

Comment:
Is it OK to use the Admin right now?

Reply:
I don't see why not; I assume it is working and has the right version of TinaCMS.

This is more a question for @jeffsee55 and the product team on when they want `TinaAdmin` to start being bundled with our starters.

Reply:
Talked about it, We are keeping admin in.